GoogleDataTransportCCTSupport/GDTCCTLibrary/GDTCCTUploader.m

```diff
@@ -174,6 +174,7 @@ - (NSString *)FLLAndCSHAPIKey {
 }
 
 - (void)uploadPackage:(GDTCORUploadPackage *)package {
+#if !TARGET_OS_WATCH
```

Comment:
The goal of GDTCORPlatform is to provide abstractions and APIs that prevent any other part of the codebase from having to have an #if TARGET_OS_XXXXX in it. Try to figure out an API, or how to use an existing API, to achieve this. We don't want TARGET_OS_XXXXXs all over the codebase. Beginning and ending activities should just happen when we call the appropriate backgroundTask APIs.